iree/modules/strings/BUILD

```diff
@@ -67,5 +67,8 @@ iree_bytecode_module(
     testonly = True,
     src = "strings_module_test.mlir",
     cc_namespace = "iree::strings_module_test",
-    flags = ["-iree-mlir-to-vm-bytecode-module"],
+    flags = [
+        "-iree-mlir-to-vm-bytecode-module",
+        "-iree-hal-target-backends=dylib-llvm-aot",
+    ],
```

Comment:
whoops, `strings_module_test` is using `"//iree/hal/vmla/registration",`... switching the compiler flags isn't sufficient here :P 

Might have missed a few other cases. Strange that the test(s) still pass...